src/classes/Standardizer/plugins/Discord/getters/getConnections.ts

```diff
@@ -0,0 +1,34 @@
+import Discord from '../Discord'
+import { Connection } from '../../../../../types/schemas'
+
+const BASE_CONNECTIONS_FOLDER = 'activity/reporting/'
+
+Discord.prototype.getConnections = async function getConnections(options) {
+  const files = await this.parser.listFiles(BASE_CONNECTIONS_FOLDER, { extensionWhitelist: ['.json'] })
+  const connections : Array<Connection> = []
+
+  for await (const file of files) {
+    const connectionFile = await this.parser.parseAsJSONL(file, options?.parsingOptions)
+    connectionFile.filter((r) => r.event_type === 'session_start')
```

Comment:
Use the filter option of JSONL parser to avoid to have the full file in RAM.

Reply:
Fixed!